externals/llvm-external-projects/torch-mlir-dialects/README.md

```diff
@@ -9,3 +9,15 @@ This project is intended to be used via LLVM's external projects setup:
 * `-DLLVM_EXTERNAL_TORCH_MLIR_DIALECTS_SOURCE_DIR={this_directory}`
 
 It depends on the `mlir` project.
+
+## TCP Dialect
+Tensor Compute Primitives (TCP) aims to be a mid-level dialect in MLIR that enables transformations that complement those in existing dialects, by supporting ops at a high-level of abstraction.
+
+TCP is being bootstrapped under Torch-MLIR Dialects.
+
+All technical discussions regarding TCP will be done in the discourse [TCP-WG](https://discourse.llvm.org/c/mlir/mlir-tcp-wg/36) category.
+### References
+* Discourse [thread](https://discourse.llvm.org/t/rfc-proposal-for-a-high-level-ml-dialect-in-mlir/64249) regarding TCP proposal.
+* TCP [proposal document](https://docs.google.com/document/d/1f3KVsXA4xm6W7gd2cKx9ThGA52XMfZKsPV3ZOlsztC4/edit?usp=sharing)
```

Comment:
Drop `?usp=sharing` from the link? I don't think it's essential. (Same for the spec draft link below).

Reply:
Removed